tutorials/dapt-curation/code/utils.py

```diff
@@ -300,12 +300,14 @@ def fuzzy_dedupe(dataset: DocumentDataset, cache: str) -> DocumentDataset:
         id_field="id",
         text_field="text",
         seed=42,
-        char_ngrams=24,
+        char_ngrams=20,
```

Comment:
Curious if these changes are needed or came from merge conflicts with the previous version of the notebook.